scripts/regressor_finder.py

```diff
@@ -262,7 +263,15 @@ def append_bug_fixing_commits(bug_id: int, type_: str) -> None:
                     append_bug_fixing_commits(bug["id"], "e")
                 continue
 
-            if defect_model.classify(bug)[0] == "defect":
+            bugs_to_classify.append(bug)
+
+        classified_bugs = []
+        if bugs_to_classify:
+            classified_bugs = defect_model.classify(bugs_to_classify)
+
+        for i, label in enumerate(classified_bugs):
```

Comment:
Nit: you can do `for i, (bug, label) in enumerate(zip(bugs_to_classify, classified_bugs)):`
so you can skip `bug = bugs_to_classify[i]`

Reply:
Thanks, I only use i for getting the bug so just removed the enumerate and kept the zip.

Reply:
Good idea